.github/workflows/ci_tests.yml

```diff
@@ -64,6 +64,10 @@ jobs:
         pip install -e .
         pip install safety
 
+    - name: List installed packages
+      run: |
+        pip list
```

Comment:
Why ddo you want this?

Reply:
I added it for debugging purposes to see what version of all packages that are installed on GitHub (which might differ from my local machine). 

It doesn't take any time, so I thought that it might be useful to keep to make it easier to spot dependency issues on GitHub. But we can also remove it again if you want.

Reply:
Your choice :)